src/Helpers/__tests__/imageupload.spec.js

```diff
@@ -0,0 +1,29 @@
+import imageupload from "../imageupload";
+import axios from "axios";
+
+jest.spyOn(axios, "post");
+
+describe("imageupload fn()", () => {
+  test("Test the imageupload functionality", () => {
+    const responsedata = {
+      data: {
```

Comment:
You can put this data in a mock file, so that you can reuse.

Reply:
Code climate complained of the same, so I have refactored the code at this point